ui/components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,53 @@
+import qs from "query-string";
+import React from "react";
+import styled from "styled-components";
+import useNavigation from "../hooks/navigation";
+import { getParentNavValue, V2Routes } from "../lib/nav";
+import Flex from "./Flex";
+import Icon, { IconType } from "./Icon";
+import Link from "./Link";
+import Text from "./Text";
+
+const CrumbLink = styled(Link)`
+  ${Text} {
+    font-size: ${(props) => props.theme.fontSizes.large};
+    color: ${(props) => props.theme.colors.neutral00};
+  }
+`;
+
+export const Breadcrumbs = () => {
+  const { currentPage } = useNavigation();
+  const parentValue = getParentNavValue(currentPage);
+
+  function pageLookup(route) {
+    const parsed = qs.parse(location.search);
```

Comment:
Blocker: Is this the global `window.location`? This will make the component hard to test. Instead, we should pull location out of the context via the `useLocation` hook that `react-router` provides.